executor/common_linux.h

```diff
@@ -5239,3 +5239,44 @@ static long syz_80211_join_ibss(volatile long a0, volatile long a1, volatile lon
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_clone3
+#include <linux/sched.h>
+#include <sched.h>
+
+#define USLEEP_FORKED_CHILD 250000 // 250 ms.
```

Comment:
Express this via these consts:
```
static uint64 syscall_timeout_ms;
static uint64 program_timeout_ms;
static uint64 slowdown_scale;
```

Do you mean `5*syscall_timeout_ms` or what?

Reply:
Set it to 3 * syscall_timeout_ms. Slowdown is already included there.

Also had to introduce an extra template parameter as those 3 variables are not available for C repros and `CALL_TIMEOUT_MS` is not a plain number.

Reply:
Found what prevented it from substituting the replacement value. Fixed now.